From 592250eeaafe2a34b08e973ac97afe389617a3e6 Mon Sep 17 00:00:00 2001
From: wangqr <wangqr@wangqr.tk>
Date: Thu, 6 Jun 2019 15:24:40 -0400
Subject: [PATCH] Remove call to wxSizer::CalcMin and wxSizer::RecalcSizes

As they are used internally by Layout and should not be called
directly.
---
 src/frame_main.cpp | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/frame_main.cpp b/src/frame_main.cpp
index b35c32960..df115b27c 100644
--- src/frame_main.cpp
+++ src/frame_main.cpp
@@ -240,8 +240,6 @@ void FrameMain::SetDisplayMode(int video, int audio) {
 	TopSizer->Show(videoBox, showVideo, true);
 	ToolsSizer->Show(audioBox, showAudio, true);
 
-	MainSizer->CalcMin();
-	MainSizer->RecalcSizes();
 	MainSizer->Layout();
 	Layout();
 
